Example/SPGDPRExampleAppUITests/SPGDPRExampleAppUITests.swift

```diff
@@ -124,6 +124,15 @@ class SPGDPRExampleAppUITests: QuickSpec {
             self.app.saveAndExitButton.tap()
             expect(self.app.privacyManager).to(disappear())
         }
+
+        it("Terms And Conditions link opens in other webview") {
```

Comment:
@vilasma it should open on Safari instead. Isn't this the case?

Reply:
@andresilveirah link open on other Safari webview only. should I add `Safari` word also?

Reply:
@vilasma I believe it'd be more explicit if we exchanged _webview_ for _Safari_ or _Browser_.